src/player/PlayerComponent.cpp

```diff
@@ -282,6 +297,13 @@ void PlayerComponent::queueMedia(const QString& url, const QVariantMap& options,
   extraArgs.insert("ad", "");
   extraArgs.insert("vd", "");
 
+  extraArgs.insert("tls-verify", "yes");
+  extraArgs.insert("stream-lavf-o", "verifyhost=" + host);
+
+#ifdef Q_OS_LINUX
+  extraArgs.insert("tls-ca-file", "/etc/ssl/cert.pem");
+#endif
```

Comment:
I'm on Linux, but I don't have this file. Could be made OPENELEC only, or dynamically by checking whether the file exists.